backup/bin/barman-parse-metrics

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+"""Usage:
+  barman-parse-metrics <path>
+"""
+
+import docopt
+import json
+import os
+import re
+import tempfile
+
+from datetime import datetime, timezone
+from dateutil import parser
+
+
+# These should come from Alex's module code
+def parse_timestamp(raw):
+    if raw and raw != "None":
+        return parser.parse(raw)
+    else:
+        return None
+
+
+def seconds_elapsed_since(timestamp):
+    if timestamp:
+        now = datetime.now()
+        return (now - timestamp).total_seconds()
+    else:
+        return None
+
+
+def clean_key(k, prefix):
+    return prefix + re.sub("[^A-Za-z0-9]", "_", k)
+
+
+def parse_size(s):
+    d = s.split(" ")
+    size_name = {"B": 0, "KiB": 10, "MiB": 20, "GiB": 30, "TiB": 40}
+    return float(d[0]) * 2**size_name[d[1]]
+
+
+def parse_last_backup_age(s):
+    last_available = parse_timestamp(s)
+    if last_available:
+        since_last_backup = seconds_elapsed_since(last_available)
+        return {
+            "barman_time_of_last_backup": last_available.timestamp(),
+            "barman_time_since_last_backup_seconds": since_last_backup,
+            "barman_time_since_last_backup_minutes": since_last_backup / 60,
+            "barman_time_since_last_backup_hours": since_last_backup / 3600,
+            "barman_time_since_last_backup_days":
+            since_last_backup / (3600 * 24)
+        }
+    else:
+        return {}
+
+
+def parse_status(path):
+    check = barman_output_as_dict(read_lines(path + '/check.txt'))
+    status = barman_output_as_dict(read_lines(path + '/status.txt'))
+    ret = {clean_key(k, "barman_check_"): int(v.startswith("OK"))
+           for (k, v) in check.items()}
+    ret["barman_ok"] = all(x == 1 for x in ret.values())
+    ret["barman_running"] = 1
```

Comment:
We were trying to distinguish between three states, but this may no longer be relevant:

* Barman is not running
* Barman is running, but an error occurred getting status from it
* Barman is running and we were able to get the status

Reply:
ok, with the changes now we'll have:

`barman_ok` - top level warning signal - everything is ok
`barman_metrics_ok` - there was any error running the metrics command
`barman_running` - barman responded as expected

We can refine later